pages/src/collection-grid/collection-grid.less

```diff
@@ -56,26 +73,45 @@
     color: white;
     -webkit-text-stroke: 1px @nav-dark-selected;
     text-shadow: -1px -1px 0 @nav-dark-selected, 1px -1px 0 @nav-dark-selected,
-      -1px 1px 0 @nav-dark-selected, 1px 1px 0 @nav-dark-selected;
+    -1px 1px 0 @nav-dark-selected, 1px 1px 0 @nav-dark-selected;
   }
+
   &-alt-small {
     font-size: 1.75rem;
   }
 
   &-column {
-    grid-template-columns: 1fr;
+    grid-template-columns: 1fr 1fr;
 
   }
+
   &-column &-item {
+    background-color: white;
+    border-radius: 0.75rem;
+    border: 1px solid #c8d6f0;
     max-width: 100%;
     flex-direction: row;
+    svg {
+      display: none;
+    }
 
+    &:hover  {
+      svg {
+        display: block;
+        fill: @primary-blue;
+      }
+      border-color: @primary-blue;
+    }
```

Comment:
I'd recommend to replace arrow SVG with a CSS alternative with a pseudo-element and border embedded svg